shell.nix

```diff
@@ -0,0 +1,26 @@
+let
+  pkgs = import
+    (
+      fetchTarball {
+        # TODO: Pin to stable nixpkgs
+        name = "nixpkgs-2021-09-28";
+        url = "https://github.com/NixOS/nixpkgs/archive/ed8c752e13ef5a217806556a96b51ca7f7fb1007.tar.gz";
+        sha256 = "03yharwv0lal286d3zy6b7kj4px111s5h3a8nar8banpnqgml7v5";
+      })
+    { };
+  poetryEnv = pkgs.poetry2nix.mkPoetryEnv {
+    python = pkgs.python38;
+    projectDir = builtins.path { path = ./.; name = "geostore"; };
+  };
+in
+pkgs.mkShell {
+  buildInputs = [
+    pkgs.nodejs-14_x
+    poetryEnv
+    pkgs.python38Packages.pip
```

Comment:
I am not sure if you need `pip` here.

Reply:
I did add it for a reason, but I can't remember now. I've removed it; we can always add it back in if necessary.

Reply:
Was is because of this error ?

```

  ModuleNotFoundError

  No module named 'virtualenv.activation.nushell'

  at <frozen importlib._bootstrap>:973 in _find_and_load_unlocked
```

Reply:
No, where do you see that?